Tests/UITests/UITests.xcodeproj/project.pbxproj

```diff
@@ -373,11 +373,11 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				OTHER_SWIFT_FLAGS = "-enable-upcoming-feature InferSendableFromCaptures";
 				SDKROOT = iphoneos;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_STRICT_CONCURRENCY = complete;
+				SWIFT_VERSION = 6.0;
```

Comment:
Wondering if we should also update the Package.swift file to move completely to Swift 6.0 now that we fixed all the crashes and could move right to Swift 6.0 by default for all things?

Reply:
We still had a compiler issue that didn't allow us to move to Swift 6 Mode on the main target (see https://github.com/swiftlang/swift/issues/76005). However, I spent some time trying to find a minimal triggering example and was successful (see https://github.com/swiftlang/swift/issues/76005#issuecomment-2466703851).

With https://github.com/StanfordSpezi/SpeziBluetooth/pull/51/commits/7c2f1eba728f4dbf188f69688fe946ba274632bb we were now able to work around the issue for now.